cmd/exec.go

```diff
@@ -176,7 +180,15 @@ func RunCommand(subcommand string, withEnvironment bool, withMonitoring bool) in
 		case err := <-waitCh:
 
 			// Send output to Cronitor and clean up after the temp file
-			outputForPing := gatherOutput(tempFile)
+			outputForPing := gatherOutput(tempFile, true)
+			var metrics map[string]int = nil
+			logLengthForPing, err := getFileSize(tempFile)
```

Comment:
looks like this err is overwriting the err on L180 that is used later on L203

Reply:
good catch

Reply:
i will rename the one on 180 -- not very defensive here

Reply:
Honestly you shouldn't have to be, in my opinion this is a failure of go structurally